src/crimson/osd/osd.cc

```diff
@@ -1292,6 +1293,79 @@ seastar::future<> OSD::handle_recovery_subreq(
     conn, std::move(m)).second;
 }
 
+vector<DaemonHealthMetric> OSD::get_health_metrics()
+{
+  LOG_PREFIX(OSD::get_health_metrics);
+  vector<DaemonHealthMetric> metrics;
+
+  const utime_t now = ceph_clock_now();
+  utime_t oldest_secs = now;
+  utime_t too_old = now;
+  too_old -= local_conf()->osd_op_complaint_time;
```

Comment:
`osd_op_complaint_time` should be added to `OSD::get_tracked_conf_keys()` and `OSD::handle_conf_change`.